packages/next/next-server/server/router.ts

```diff
@@ -132,11 +132,13 @@ export default class Router {
               requireBasePath: false,
               match: route('/:path*'),
               fn: async (checkerReq, checkerRes, params, parsedCheckerUrl) => {
-                const { pathname } = parsedCheckerUrl
+                let { pathname } = parsedCheckerUrl
+                pathname = pathname?.replace(/\/$/, '') || '/'
```

Comment:
is it possible to reuse https://github.com/vercel/next.js/blob/681fbbd04d217ce2ff90d367788e3596131a8b2b/packages/next/client/normalize-trailing-slash.ts#L4 ?

Reply:
Good catch, updated!